### PR TITLE
Improve docker doc

### PR DESCRIPTION
**Pull request by sharils**
This makes login work and also saves config across invocations. Besides this doc fix, we could also support whalebrew.

### CODE REVIEW

**Review by UmanShahzad — COMMENTED** [review]
I think `--rm` makes a lot of sense for something like this but the volume part is a bit questionable, we should leave that out of the official README.md, which is just showing a demonstration command, not too opinionated.

**Comment by sharils** [comment]
I was thinking about that too. How about separate into 2 commands, one has `-it`, which is required for login. The other command shows how to save the volume, otherwise a user has to read the source to find the path and instruct them how to change the `-v` param to store to wherever they want.

**Comment by UmanShahzad** [comment]
Yeah let's do that, combine `--it` and `--rm` into one command, and a separate volume explanation.

**Comment by sharils** [comment]
I had to force update because I made the commits atomic.

File: README.md [hunk 1]
```diff
@@ -77,7 +77,15 @@ iwr -useb https://github.com/ipinfo/cli/releases/download/ipinfo-2.9.0/windows.p
 ### Docker
 
 ```bash
-docker run ipinfo/ipinfo:2.9.0
+docker run --rm -it ipinfo/ipinfo:2.9.0
+```
+
+To save the ipinfo config, add `-v "/path_to_config:/root/.config/ipinfo"`. For
```
Comment (UmanShahzad) [inline README.md:83]:
```suggestion
To save the CLI's config, add `-v "/path_to_config:/root/.config/ipinfo"`. For
```

**Review by UmanShahzad — APPROVED** [review]

**Pull request merged**